README.markdown

```diff
@@ -115,6 +131,30 @@ One can apply a "sort" constraint, which analyzes documents in the specified ord
 
     $ mongo test --eval "var collection = 'users', sort = { updated_at : -1 }" variety.js
 
+### Output formats ###
```

Comment:
In keeping with the general grammatical style of the README, could we capitalize all major words, like one would do for a title?